packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -133,14 +148,23 @@ const askApplicationExtensibiltyQuestions = (appExtensions) => {
                 'you will NO LONGER be able to consume upgrades from NPM. All changes\n' +
                 'made to the extracted code will be YOUR RESPONSIBILITY.\n' +
                 '\n' +
-                'Do you want to proceed with extracting the Application Extension code?',
+                'Do you want to proceed with extracting the Application Extensions code?',
             type: 'confirm',
             default: false,
             when: (answers) => answers.project.useAppExtensibility === true
         }
     ]
 }
 
+const APPLICATION_EXTENSION_QUESTIONS = [
+    {
+        name: 'project.extensionName',
+        message:
+            'What is the name of your Application Extension? (The prefix "extension-" will be added to the name.)',
+        validate: validProjectName
```

Comment:
The project validation doesn't apply to extensions here because the project name is being used for mrt and it have limitations on length. 

Also I thought we talked about making the validator ensure that the entered value was in the form `@{namespace}/extension-{package-name}` where the namespace is optional. 

I think the following regex will validate if the user input follows that requirement.

```
^(@[a-zA-Z0-9-_]+/)?extension-[a-zA-Z0-9-_]+$
```